desktop/src/main/java/bisq/desktop/main/dao/economy/supply/SupplyView.java

```diff
@@ -63,16 +70,26 @@
 import java.time.temporal.TemporalAdjusters;
 
 import java.util.Comparator;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+import java.util.Spliterators.AbstractSpliterator;
+
+import java.util.function.BiFunction;
+import java.util.function.Supplier;
+import java.util.function.Consumer;
+
+import javafx.collections.ListChangeListener;
 
 import static bisq.desktop.util.FormBuilder.addTitledGroupBg;
 import static bisq.desktop.util.FormBuilder.addTopLabelReadOnlyTextField;
+import static bisq.desktop.util.FormBuilder.addSlideToggleButton;
```

Comment:
Do you use IntelliJ for development? If yes, please use the formatting functionality that auto sorts and groups the different imports.

Reply:
I use vim and I do formatting and similar tasks manually. I'll ask for suggestions on #dev.